trm/lookup.tex

```diff
@@ -0,0 +1,25 @@
+\begin{description}
+	\item[\underline{Selector:}] none
```

Comment:
I would have expected IOMF selector or so. I haven't looked in detail but it seems that every row will spawn a lookup to WCP so I get that there may be no need for a selector. A bit of documentation would be nice either way .